api/rest_client/api_usage_examples.rest

```diff
@@ -1,76 +1,77 @@
 # This file contains example HTTP calls and it also the configuration
 # file for the VSCode plugin "REST Client" https://open-vsx.org/extension/humao/rest-client. 
 
-@API_SERVER = http://localhost:5001
-
-# Bearer Token starts with 'ey...', no quotes
-@AUTH_BEARER_TOKEN = ey...
-
+#Add the following to the .env file in this directory:
 
+#API_SERVER=http://localhost:5001/
+#AUTH_BEARER_TOKEN = ey...
+#USER_ID = <uuid>
+#USER_EMAIL = <email>
+#USER_PASSWORD = <password>
 
 # Health Check
-{{API_SERVER}}
+{{$dotenv API_SERVER}}
 
 # ------------------------------------------------------------------------------
 # Auth Endpoint
 # ------------------------------------------------------------------------------
 
 ### Auth Health Check
 # Returns a health check message for the auth endpoint.
-{{API_SERVER}}/auth/
+{{$dotenv API_SERVER}}/auth/
 
 ### auth_current_user()
 # Current User. You can get the AUTH_BEARER_TOKEN from the browser's console by looking at 
 # a request to graphql-engine. It is only as long-lived as your browser session.
 # Returns the current user's Cognito username, email, Hasura claims, and the full decrypted JWT.
-{{API_SERVER}}/auth/current_user
-Authorization: Bearer {{AUTH_BEARER_TOKEN}}
+{{$dotenv API_SERVER}}/auth/current_user
+Authorization: Bearer {{$dotenv AUTH_BEARER_TOKEN}}
```

Comment:
What do you think about adding an example or two that tests to make sure an invalid token is rejected?